drivers/i2s/i2s_stm32_sai.c

```diff
@@ -409,7 +409,6 @@ static int i2s_stm32_sai_initialize(const struct device *dev)
 static void dma_callback(const struct device *dma_dev, void *arg, uint32_t channel, int status)
 {
 	DMA_HandleTypeDef *hdma = arg;
-
```

Comment:
Please restore the empty line.
It's expected between variable definitions and instructions (which `ARG_UNUSED()` is when in debug mode).